### ISSUE TITLE
error when using `-o` and `-n`

### ISSUE THREAD

**Issue by YehudaKremer** [YehudaKremer/msix#98]
Thank you, when running without `-o` nor `-n` option it works fine. 

But when using the options `-o` and `-n` to specify the output path, i get an error:

```
RangeError (start): Invalid value: Not in inclusive range 0..48: -1
#0      RangeError.checkValidRange (dart:core/errors.dart:350:7)
#1      _StringBase.substring (dart:core-patch/string_patch.dart:400:27)
#2      Msix.create (package:msix/msix.dart:37:10)
<asynchronous suspension>
pub finished with exit code 255
```

(corresponding line tries to find `/build/windows` which is not in my designated output path, so maybe it has problems there?)

_Originally posted by @khainke in https://github.com/YehudaKremer/msix/issues/74#issuecomment-1036196933_

**Comment by YehudaKremer** [comment]
Hello @khainke

fixed in version [3.0.0-dev.9](https://pub.dev/packages/msix/versions/3.0.0-dev.9),
thank you fox reporting those bugs 👍

**Comment by khainke** [comment]
Thank you for the fix, works fine now for me.
